descriptors/ModuleDescriptor-template.json

```diff
@@ -4,35 +4,24 @@
   "provides": [
     {
       "id": "rtac",
-      "version": "1.4",
+      "version": "2.0",
       "handlers": [
         {
-          "methods": ["GET"],
-          "pathPattern": "/rtac",
-          "permissionsRequired": ["rtac.holdings.collection.get"]
-        },
-        {
-          "methods": ["GET"],
-          "pathPattern": "/rtac/{id}",
```

Comment:
I didn't find any dependencies on RTAC, except in EDGE-RTAC.

Reply:
My concern isn't with FOLIO module dependencies, but with unknown dependencies, which I feel that the FOLIO community tends to not consider. Like I said, this module has been deployed for a long time and implementers could have custom scripts, tools or modules that use the public APIs in mod-rtac. It would be nice to have a deprecation period before APIs disappear. However, if supporting the old endpoint with the new code is not feasible, then we can skip this.

Reply:
According to @mtraneis, as long as we support the existing API in edge-rtac, which we do, we'll be fine. Please confirm. @mtraneis .

Reply:
@mrdpopov @mreno-EBSCO's may be correct. I don't know if anyone in the community or outside of the community is using it :(  To be on the safe side we may need to leave it there for a while or mark it deprecated